### DISCUSSION TITLE
Display screenshots based on `prefers_color_scheme`

### DISCUSSION THREAD

**Discussion by meisenzahl** [elementary/appcenter#1598] [Ideas]
I had thought about whether it would be useful to be able to annotate the screenshots in AppData as to whether they are for `prefers_color_scheme` `dark`.

```xml
<screenshot type="default" prefers_color_scheme="dark">
    <image>https://raw.githubusercontent.com/manexim/typewriter/master/data/screenshots/000-dark.png</image>
</screenshot>
```

Then AppCenter could display the appropriate screenshots based on the current setting:

# no-preference

![image](https://user-images.githubusercontent.com/10796736/129435857-da5f461a-da77-40e5-95a7-0b5a88ce7641.png)

# dark

![image](https://user-images.githubusercontent.com/10796736/129435861-7d22bd18-a5fa-4330-ba7d-6c12d86d7ac1.png)

**Comment by danirabbit** [comment]
Yeah I think we'd have to propose this upstream to the appstream spec, but I think this is a good idea :+1:

**Reply by danirabbit** [reply]
Closing because this is actually done now :)
